data/json/bionics.json

```diff
@@ -613,31 +613,31 @@
     "id": "bio_metabolics",
     "type": "bionic",
     "name": "Metabolic Interchange",
-    "description": "Your digestive system and power supply are interconnected.  Bionic energy is replenished at the cost of increasing hunger.  Can be toggled on and off at will.",
+    "description": "Your digestive system and power supply are interconnected.  Bionic energy is replenished at the cost of decreasing nutrition.  Can be toggled on and off at will.",
```

Comment:
MI draws power from calorie storage, so you could say that it's burning sugar pulled from your blood stream instead of being connected to your digestive system.